projects/Ethersync/programs/ethersync/tests/basic.nix

```diff
@@ -0,0 +1,79 @@
+{
+  sources,
+  lib,
+  pkgs,
+  ...
+}:
+
+{
+  name = "Ethersync";
+
+  interactive.sshBackdoor.enable = true;
+
+  nodes =
+    let
+      config = {
+        imports = [
+          sources.modules.ngipkgs
+          sources.modules.programs.ethersync
+          sources.examples.Ethersync.basic
+        ];
+      };
+    in
+    {
+      server = config // {
+        networking.firewall.allowedTCPPorts = [ 4242 ];
+        networking.interfaces.eth1.ipv4.addresses = lib.mkForce [
+          {
+            address = "192.168.1.1";
+            prefixLength = 24;
+          }
+        ];
+      };
+      client = config // {
+        services.getty.autologinUser = "root";
+        networking.interfaces.eth1.ipv4.addresses = lib.mkForce [
+          {
+            address = "192.168.1.2";
+            prefixLength = 24;
+          }
+        ];
+      };
+    };
+
+  testScript =
```

Comment:
Adding comments to clarify what things do and why we're adding them would be nice.